net/keepalived/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=keepalived
-PKG_VERSION:=2.2.7
+PKG_VERSION:=2.2.8
 PKG_RELEASE:=10
```

Comment:
```suggestion
PKG_RELEASE:=1
```

Reply:
@BKPepe Thanks for the hint.I have also just seen :+1: